### PR TITLE
Use latest vanilla docs theme

### PR DESCRIPTION
**Pull request by grahambancroft**
## Done

Update to use latest vanilla docs theme

## QA

This update breaks a lot of things so want this to be its own PR and I’ll fix everything up in another PR but please check that it’s now using the latest vanilla docs theme

https://github.com/vanilla-framework/vanilla-docs-theme

### CODE REVIEW

File: static/sass/_cards.scss [hunk 1]
```diff
@@ -1,6 +1,6 @@
 @charset 'utf-8';
 @import 'global-settings';
-@import 'docs-vanilla-theme/scss/theme';
+@import 'vanilla-docs-theme/scss/theme';
```
Comment (anthonydillon) [inline static/sass/_cards.scss]: Do you need to import vanilla here as its imported in global.scss too

**Review by anthonydillon — COMMENTED** [review]
Question in code.

File: static/sass/_cards.scss (hunk 1 above)
Comment (grahambancroft) [reply static/sass/_cards.scss]: You’re right, it’s not needed, I’ve removed it

**Review by anthonydillon — APPROVED** [review]
Thanks :+1:

**Pull request merged**